<?xml version="1.0"?>
<launch>

    <arg name="in_odom_topic" default="/odom"/>
    <arg name="out_odom_topic" default="/odom/throttled"/>
    <arg name="distance_threshold" default="0.05"/>
    <arg name="rotation_threshold" default="0.1"/>

    <node pkg="turtlebot_interface" name="odom_throttle_node" type="odom_throttle_node" output="screen">
        <param name="in_odom_topic" value="$(arg in_odom_topic)"/>
        <param name="out_odom_topic" value="$(arg out_odom_topic)"/>
        <param name="distance_threshold" value="$(arg distance_threshold)"/>
        <param name="rotation_threshold" value="$(arg rotation_threshold)"/>
    </node>

</launch>
